components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -70,16 +79,20 @@ export default class PostProfilePicture extends React.PureComponent {
             user,
             isBot,
         } = this.props;
+
         const isSystemMessage = PostUtils.isSystemMessage(post);
         const fromWebhook = PostUtils.isFromWebhook(post);
+
         if (isSystemMessage && !compactDisplay && !fromWebhook && !isBot) {
             return <MattermostLogo className='icon'/>;
         }
-
         const fromAutoResponder = PostUtils.fromAutoResponder(post);
 
         const hasMention = !fromAutoResponder && !fromWebhook;
-        const src = this.getProfilePicSrcForPost(fromAutoResponder, fromWebhook);
+        const profileSrc = this.getProfilePictureURL();
+        const src = this.getPostIconURL(profileSrc, fromAutoResponder, fromWebhook);
+
+        const isEmoji = post.props && !isEmpty(post.props.override_icon_emoji);
```

Comment:
Can you just check if `post.props.override_icon_emoji !== ''` here? We don't really need to start using lodash just for this